bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveWorkItem.java

```diff
@@ -239,18 +239,17 @@ private String subjectPrefix() {
         return ret.toString();
     }
 
+    private String mboxFile() {
+        return bot.codeRepo().name() + "/" + pr.id() + ".mbox";
+    }
+
     @Override
     public Collection<WorkItem> run(Path scratchPath) {
         var path = scratchPath.resolve("mlbridge");
-        var archiveRepo = materializeArchive(path);
-        var mboxBasePath = path.resolve(bot.codeRepo().name());
 
         var sentMails = new ArrayList<Email>();
-        try {
-            var archiveContents = Files.readString(mboxBasePath.resolve(pr.id() + ".mbox"), StandardCharsets.UTF_8);
-            sentMails.addAll(Mbox.splitMbox(archiveContents, bot.emailAddress()));
-        } catch (IOException ignored) {
-        }
+        var archiveContents = bot.archiveRepo().fileContents(mboxFile(), bot.archiveRef());
+        archiveContents.ifPresent(s -> sentMails.addAll(Mbox.splitMbox(s, bot.emailAddress())));
```

Comment:
So this adds the old contents to sentMails, and if there is none, we just ignore adding it? I needed some time to figure out why it was safe doing nothing if archiveContents was empty. Perhaps a comment about this?

Reply:
I didn't reflect on that as it's the same logic as before, only before it was handled with the ignored `IOException`, but I can add a comment.